acceptancetests/deploy_stack.py

```diff
@@ -202,7 +202,7 @@ def _get_token(remote, token_path="/var/run/dummy-sink/token"):
         if e.returncode != 1:
             raise
         return ""
-    return token_pattern.match(contents.decode("utf-8")).group(1)
+    return token_pattern.match(contents).group(1)
```

Comment:
`remote.cat` decodes the bytes it gets back internally so contents is already a string and needs no decoding.